tornado_rest_client/test/integration_api.py

```diff
@@ -70,7 +73,7 @@ class IntegrationRestConsumer(testing.AsyncTestCase):
     def integration_base_get(self):
         httpbin = HTTPBinRestConsumer()
         ret = yield httpbin.http_get()
-        self.assertIn('DOCTYPE', ret)
+        self.assertIn(six.b('DOCTYPE'), ret)
```

Comment:
both python2 and python3 have byte/unicode string literals.  `b'this is a byte string'` `u'this is a unicode string'`